speech_assets/SampleUtterances.txt

```diff
@@ -588,15 +972,73 @@ ShuffleVideoPlaylist shuffle {VideoPlaylist} show playlist
 ShuffleVideoPlaylist shuffle {VideoPlaylist} video playlist
 Shutdown shut down
 Shutdown shut down system
-Skip listen to next
+Skip listen to next item
+Skip listen to next one
 Skip listen to next song
-Skip next
-Skip next song
-Skip play next
```

Comment:
Fair enough

"Alexa, ask kodi to next"
"Alexa, tell kodi previous"

Testing all the prev and skip intents does highlight some clashes with the audio stream controls.

"to previous" = AudioStreamPrevious
"to next" = Skip
"previous" = AMAZON.PreviousIntent
"next" = AMAZON.NextIntent
"next track" = Skip